lib/gears/shape.lua

```diff
@@ -331,6 +331,7 @@ end
 -- @tparam[opt=math.min(width  height) / 2)] number radius The radius
 function module.circle(cr, width, height, radius)
     radius = radius or math.min(width, height) / 2
+    cr:move_to(width/2+radius, height/2)
```

Comment:
What happens without this? Looking at cairo's implementation (`_cairo_default_context_arc()`), the first thing that it does is a `line_to` to the start point of the arc. If you just want to get rid of this `line_to`, I would suggest `cr:new_sub_path()` instead (easier to read/understand).